Common/src/config_structure.cpp

```diff
@@ -6301,6 +6318,19 @@ void CConfig::SetOutput(unsigned short val_software, unsigned short val_izone) {
     cout << "Residual minimum value: 1e" << MinLogResidual << "." << endl;
     cout << "Cauchy series min. value: " << Cauchy_Eps << "." << endl;
     cout << "Number of Cauchy elements: " << Cauchy_Elems << "." << endl;
+    if(Cauchy_Elems <1){
+        SU2_MPI::Error(to_string(Cauchy_Elems) + string(" Cauchy elements are no viable input. Please check your configuration file."), CURRENT_FUNCTION);
```

Comment:
It seems most of the code has 4 space indentation, make that 2 space (on c++ files) to be consistent with the rest of the code.